cyipopt/tests/unit/test_scipy_ipopt_trust_constr.py

```diff
@@ -443,178 +446,97 @@ def hess(x, v):
-
-        result = minimize(prob.fun, prob.x0,
-                          method='trust-constr',
-                          jac=prob.grad, hessp=hessp,
-                          bounds=prob.bounds,
-                          constraints=prob.constr)
-
-        if prob.x_opt is not None:
-            assert_array_almost_equal(result.x, prob.x_opt, decimal=2)
-
-        # gtol
-        if result.status == 1:
-            assert_array_less(result.optimality, 1e-8)
-        # xtol
-        if result.status == 2:
-            assert_array_less(result.tr_radius, 1e-8)
-
-            if result.method == "tr_interior_point":
-                assert_array_less(result.barrier_parameter, 1e-8)
-        # max iter
-        if result.status in (0, 3):
-            raise RuntimeError("Invalid termination condition.")
 
     def test_args(self):
         prob = MaratosTestArgs("a", 234)
 
         result = minimize(prob.fun, prob.x0, ("a", 234),
-                          method='trust-constr',
-                          jac=prob.grad, hess=prob.hess,
+                          method=None,
+                          jac=prob.grad,
                           bounds=prob.bounds,
                           constraints=prob.constr)
 
         if prob.x_opt is not None:
             assert_array_almost_equal(result.x, prob.x_opt, decimal=2)
 
-        # gtol
-        if result.status == 1:
-            assert_array_less(result.optimality, 1e-8)
-        # xtol
-        if result.status == 2:
-            assert_array_less(result.tr_radius, 1e-8)
-            if result.method == "tr_interior_point":
-                assert_array_less(result.barrier_parameter, 1e-8)
-        # max iter
-        if result.status in (0, 3):
-            raise RuntimeError("Invalid termination condition.")
-
-    def test_raise_exception(self):
-        prob = Maratos()
-        message = "Whenever the gradient is estimated via finite-differences"
-        with pytest.raises(ValueError, match=message):
-            minimize(prob.fun, prob.x0, method='trust-constr', jac='2-point',
-                     hess='2-point', constraints=prob.constr)
+        assert result.success
 
     def test_issue_9044(self):
         # https://github.com/scipy/scipy/issues/9044
         # Test the returned `OptimizeResult` contains keys consistent with
         # other solvers.
 
-        def callback(x, info):
-            assert_('nit' in info)
-            assert_('niter' in info)
-
         result = minimize(lambda x: x**2, [0], jac=lambda x: 2*x,
-                          hess=lambda x: 2, callback=callback,
-                          method='trust-constr')
+                          hess=lambda x: 2, method=None)
         assert_(result.get('success'))
-        assert_(result.get('nit', -1) == 1)
+        assert result.get('nit', -1) == 0
 
-        # Also check existence of the 'niter' attribute, for backward
-        # compatibility
-        assert_(result.get('niter', -1) == 1)
```

Comment:
I think this was because `trust-constr` used to have the wrong name for this attribute. In any case, not needed here.